client/my-sites/backup/backup-storage-space/index.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import { sprintf } from '@wordpress/i18n';
+import { Card, ProgressBar } from '@automattic/components';
+import React, { FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import {
+	BackupStorageSpaceUpsell,
+	BackupStorageSpaceUpsellOptions,
+} from './backup-storage-space-upsell';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const upsellLimit1 = 0.6;
+const upsellLimit2 = 0.85;
+
+const progressBarWarnings: Record< BackupStorageSpaceUpsellOptions, string > = {
+	no_upsell: 'no-warning',
+	first_upsell: 'yellow-warning',
+	second_upsell: 'red-warning',
+	out_of_storage: 'red-warning',
+};
+
+type Props = Record< string, never >;
+
+export const BackupStorageSpace: FunctionComponent< Props > = () => {
+	const storageLimit = 200;
+	const usedStorage = 200;
+
+	const translate = useTranslate();
+
+	const usedStorageFraction = usedStorage / storageLimit;
+
+	let upsellOption: BackupStorageSpaceUpsellOptions = 'no_upsell';
+	if ( usedStorageFraction >= upsellLimit1 ) {
+		upsellOption = 'first_upsell';
+	}
+	if ( usedStorageFraction >= upsellLimit2 ) {
+		upsellOption = 'second_upsell';
+	}
+	if ( usedStorage >= storageLimit ) {
+		upsellOption = 'out_of_storage';
+	}
+
+	const showUpsell = upsellOption !== 'no_upsell';
+
+	const progressBarWarning = progressBarWarnings[ upsellOption ];
+
+	// TODO: account for MB/GB and translate once API data is available
+	const title = sprintf( '%1$sGB of %2$sGB used', usedStorage, storageLimit );
+
+	return (
+		<Card className="backup-storage-space">
+			<div className="backup-storage-space__progress-bar-container">
+				<div className="backup-storage-space__progress-heading">
+					{ translate( 'Storage space' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **134** times:
`translate( 'Storage Space' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/backup-storage-space-component)